tests/test_physics.py

```diff
@@ -0,0 +1,46 @@
+from matplotlib.dates import num2timedelta
+import yaml
+import os
+
+import xarray as xr
+
+import torch
+from credit.physics_core import compute_pressure_on_mlevs
+
+TEST_FILE_DIR = "/".join(os.path.abspath(__file__).split("/")[:-1])
+CONFIG_FILE_DIR = os.path.join("/".join(os.path.abspath(__file__).split("/")[:-2]),
+                      "config")
+
+def test_pressure_on_mlevs():
+    # testing on derecho
+    # level_info = xr.open_dataset("/glade/u/home/wchapman/MLWPS/DataLoader/ERA5_Lev_Info.nc")
```

Comment:
If you want to have derecho specific testing functionality, I would add an if statement that can check the host name. Otherwise, please delete.

Reply:
Given that it is merely loading data, just delete.